src/lbaf/IO/lbsMeshBasedVisualizer.py

```diff
@@ -316,8 +357,9 @@ def create_object_mesh(self, phase: Phase, object_mapping: set):
         # Assign edge volume values
         self.__logger.debug(f"\tedges:")
         for e in range(n_e):
-            v_arr.SetTuple1(e, edges.get(index_to_edge[e], float("nan")))
-            self.__logger.debug(f"\t{e} {index_to_edge[e]}): {v_arr.GetTuple1(e)}")
+            i2e =index_to_edge[e]
```

Comment:
Do you want to fix the spacing so that it doesn't end up a diff on the next PR? ;-)